doc/guides/tips-tricks-and-troubleshooting.md

```diff
@@ -0,0 +1,74 @@
+# Tips, Tricks & Troubleshooting
```

Comment:
This title seems a bit too vague. I'm not fond of junk-drawer documents (that is to say, a document you throw information in when you don't know where else to put it). So I'd prefer a specific title.